test/fixtures/bundle-app-src-map.js

```diff
@@ -0,0 +1,31 @@
+// Generated by purs bundle 0.13.6
```

Comment:
If we reuse the fixture that we already have then we would not need to add any new ones in this PR :slightly_smiling_face:

Reply:
Hmm... The problem is that the ".js" file that is generated with "--source-maps" flag appends the line below in the file:
`//# sourceMappingURL=<name-of-source-map-file>.js.map`

Here: https://github.com/Woody88/spago/blob/master/test/fixtures/bundle-app-src-map.js#L31

and here: https://github.com/Woody88/spago/blob/master/test/fixtures/bundle-module-src-map.js#L30

The test will fail with the ones that we already have. 

I noticed that added an extra file that is not needed, bundle-app-src-map.js.map, I will remove it. My bad...

Reply:
Right. The reason why I'd like to reuse these files is that we have some code that automatically upgrades fixtures whenever a new purescript version is out (because the version number of the compiler appears several times in these files), here:
https://github.com/spacchetti/spago/blob/a35d100d0aef5a5fa76cc18ff2a0285bd094d58f/app/Curator.hs#L296-L310

So either:
- we reuse the fixture files and append that magic string when checking the fixtures
- ..or patch the new fixture as the existing ones in the code above

Reply:
Right, I see. Thanks for showing me this piece of code. Hmm.. I don't want to do too much processing in the test file because it seems like the flow that you guys have been following is to call functions from the Utils.hs module. I am not sure if the steps of appending the magic string belong in that module. Anyhow, just so we can get this going, I will go with your first suggestion!

Reply:
Sorry if I didn't make this clearer, but my preference would be instead to just add two lines to do the `sed` thing in the code above, so that you don't have to change the test setup at all from what it is right now